.github/workflows/link-check-all.yml

```diff
@@ -2,10 +2,50 @@ name: Check all links in the repository
 on:
   workflow_call:
     inputs:
+      failsOnly:
+        type: "boolean"
+        required: false
+        description: "If true, only reports failed links"
+        default: true
+      outputFile:
+        type: "string"
+        required: false
+        description: "The file the link check report will be written to before publishing"
+        default: "link-check-report.md"
       config:
         type: 'string'
         description: 'The path to the link-check config file'
         default: 'config/link-check/config.yml'
+      owner:
+        description:
+          The owner of repository to operate on if the event doesn't include it
+        required: false
+        type: string
+        default: '${{ github.event.repository.owner.login }}'
+      repo:
+        description:
+          The repository to operate on if the event doesn't include it
+        required: false
+        type: string
+        default: '${{ github.event.repository.name }}'
+      label:
+        description:
+          The label that will be used to identify existing link-check reports to
+          edit
+        required: false
+        type: string
+        default: link-check-all
+      labels:
+        description:
+          The labels that will be assigned to issues created by this workflow
+        required: false
+        type: string
+        default: link-check-all
+      title:
+        description: The title of the issue created by this workflow
+        required: false
+        type: string
+        default: 'Link Checker Report'
```

Comment:
These are from the download link checker action, mostly needed to find existing Issues